src/app/collective-rewards/utils/chartUtils.ts

```diff
@@ -0,0 +1,14 @@
+export const toDate = (d: Date | number | string): Date => (d instanceof Date ? d : new Date(d))
```

Comment:
not sure this function offers enough value. if you think so, it can be simplified to
```suggestion
export const toDate = (d: Date | number | string): Date => new Date(d)
```